src/main/java/com/mojang/datafixers/types/JsonOps.java

```diff
@@ -86,6 +86,14 @@ public JsonElement createNumeric(final Number i) {
         return new JsonPrimitive(i);
     }
 
+    @Override
+    public Optional<Boolean> getBooleanValue(final JsonElement input) {
```

Comment:
The default logic for getting the boolean value falls back to the number value, but this one doesn't. Also, `getNumberValue` will convert boolean to a numeric too; either they should be separated completely (which I think is less convenient while not adding much in terms of error prevention), or this method should fall back to the number value too. :)

Reply:
Yeah, that's a good idea. I made the `getBooleanValue` implementation fall back to the numeric value in f488367.

Reply:
Or just stop overriding this method and use the default logic at https://github.com/Mojang/DataFixerUpper/pull/42/files#diff-10c12d1a0860902cefb4c301ccbb9088R71
 which is already correct